public/services/NotificationService.ts

```diff
@@ -25,6 +26,18 @@ export default class NotificationService {
     this.httpClient = httpClient;
   }
 
+  getServerFeatures = async (): Promise<Array<String>> => {
+    try {
+      const response = await this.httpClient.get(
+        NODE_API.GET_AVAILABLE_FEATURES
+      );
+      return response.allowed_config_type_list as Array<String>;
+    } catch (error) {
+      console.error('error fetching available features', error);
+      return null;
```

Comment:
Are there any concerns with receiving a null pointer exception somewhere by returning `null` here? Would it be worthwhile to instead return `{}`?

Reply:
good point, that would be more robust.